db/const.txt

```diff
@@ -668,9 +669,11 @@ RC_Demon	6
 RC_DemiHuman	7
 RC_Angel	8
 RC_Dragon	9
-RC_Boss	10
-RC_NonBoss	11
-RC_NonDemiHuman	12
+RC_Player	10
+RC_Boss	11
+RC_NonBoss	12
+RC_NonDemiHuman	13
```

Comment:
The problem is that if you have a script that use NonDemiHuman, you expect its behavior not to change (i.e. to still not target players). Since players aren't demihuman anymore, I'd feel more confident with a backward-incompatible change (so that it throws an error at parse time), than a subtle behavior change (so that it won't be noticeable)